### PR TITLE
Support training Stanford NER model

### PR DESCRIPTION
**Pull request by neumannm**
Here is my initial proposal for including a component for training a NER model with Stanford CoreNLP. See Issue #1000 .

### CODE REVIEW

File: dkpro-core-stanfordnlp-gpl/src/main/java/de/tudarmstadt/ukp/dkpro/core/stanfordnlp/StanfordNamedEntityRecognizerTrainer.java [hunk 1]
```diff
@@ -0,0 +1,248 @@
...
+        getLogger().info("Conversion process complete.");
+    }
+
+    /*
+     * Taken from Conll2003Writer and modified for the task at hand.
+     */
+    private void convert(JCas aJCas, PrintWriter aOut)
+    {
+        // temp variable to limit training file in size for tests
+        int i = 30;
```
Comment (reckart) [inline dkpro-core-stanfordnlp-gpl/src/main/java/de/tudarmstadt/ukp/dkpro/core/stanfordnlp/StanfordNamedEntityRecognizerTrainer.java]: This variable probably shouldn't be in there anymore, right?

File: dkpro-core-stanfordnlp-gpl/src/test/java/de/tudarmstadt/ukp/dkpro/core/stanfordnlp/StanfordNamedEntityRecognizerTrainerTest.java [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+        ConfigurationParameterFactory.setParameter(testReader,
+                Conll2002Reader.PARAM_READ_NAMED_ENTITY, true);
+        List<Span<String>> expected = EvalUtil.loadSamples(testReader, NamedEntity.class, ne -> {
+            return ne.getValue();
+        });
+        System.out.printf("Expected samples: %d%n", expected.size());
+
+        Result results = EvalUtil.dumpResults(targetFolder, expected, actual);
+
+        // TODO how to determine the expected results beforehand?
```
Comment (reckart) [inline dkpro-core-stanfordnlp-gpl/src/test/java/de/tudarmstadt/ukp/dkpro/core/stanfordnlp/StanfordNamedEntityRecognizerTrainerTest.java]: Let it run once, then put the values that the classifier outputs here. This check is mainly to see a) the code works b) to notice if any changes to the code affect the results.

File: dkpro-core-stanfordnlp-gpl/src/main/java/de/tudarmstadt/ukp/dkpro/core/stanfordnlp/StanfordNamedEntityRecognizerTrainer.java (hunk 1 above)
Comment (neumannm) [inline dkpro-core-stanfordnlp-gpl/src/main/java/de/tudarmstadt/ukp/dkpro/core/stanfordnlp/StanfordNamedEntityRecognizerTrainer.java]: Sure, sorry for that. I just removed it.

File: dkpro-core-stanfordnlp-gpl/src/test/java/de/tudarmstadt/ukp/dkpro/core/stanfordnlp/StanfordNamedEntityRecognizerTrainerTest.java (hunk 2 above)
Comment (neumannm) [inline dkpro-core-stanfordnlp-gpl/src/test/java/de/tudarmstadt/ukp/dkpro/core/stanfordnlp/StanfordNamedEntityRecognizerTrainerTest.java]: Ok thanks for explaining. I did exactly so, but I suspect the values will change now that I removed the test variable that restricted the training file to 30 sentences. I will have to see if it even runs on my machine now with the whole training file (will need a lot of memory), and if it does, I will adjust the numbers accordingly.

**Pull request merged**